Add YouTube playlist support

mathur opened the pull request:
Signed-off-by: Rohan Mathur rohan@rmathur.com

mathur committed:
Add YouTube playlist support

Signed-off-by: Rohan Mathur <rohan@rmathur.com>

mathur commented:
NOTE: I haven't been able to test this yet since I can't seem to get beats running on my local machine just yet due to some weird config error.

colegleason commented:
Please test this when you can. HAve you modified the `backend` config
parameter

On Mon, Mar 16, 2015 at 12:49 PM Rohan Mathur <notifications@github.com>
wrote:

> NOTE: I haven't been able to test this yet since I can't seem to get beats
> running on my local machine just yet due to some weird config error.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acm-uiuc/beats/pull/54#issuecomment-81838978>.
>

mathur committed:
Fix error when initializing database

When running db.py for the first time, you get:

Traceback (most recent call last):
  File "db.py", line 7, in <module>
    import art
  File "/home/rohan/Projects/beats/art.py", line 8, in <module>
    ART_DIR = config.get('Artwork', 'art_path')
  File "/usr/lib/python2.7/ConfigParser.py", line 330, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'Artwork'

Add the proper section to beats.cfg.sample

Signed-off-by: Rohan Mathur <rohan@rmathur.com>

mathur committed:
Add lxml dependency to requirements.txt

Re-run pip install to grab lxml

Signed-off-by: Rohan Mathur <rohan@rmathur.com>
